common/djangoapps/course_modes/tests/test_signals.py

```diff
@@ -0,0 +1,89 @@
+"""
+Unit tests for the course_mode signals
+"""
+
+from datetime import datetime, timedelta
+from mock import patch
+
+import ddt
+from pytz import UTC
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from course_modes.models import CourseMode
+from course_modes.signals import _listen_for_course_publish
+
+
+@ddt.ddt
+class CourseModeSignalTest(ModuleStoreTestCase):
+    """
+    Tests for the CourseMode model
```

Comment:
This docstring should indicate that the tests are for signals related to the `CourseMode` model, since we already have tests for the model itself.